bin/alks-iam-createtrustrole

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+'use strict';
+
+process.title = 'ALKS';
+
+var program   = require('commander'),
+    _         = require('underscore'),
+    clc       = require('cli-color'),
+    alks      = require('alks-node'),
+    Iam       = require('../lib/iam'),
+    utils     = require('../lib/utils'),
+    Developer = require('../lib/developer'),
+    config    = require('../package.json');
+
+var logger       = 'iam-createtrustrole',
+    roleNameDesc = 'alphanumeric including @+=._-',
+    trustArnDesc = 'arn:aws|aws-us-gov:iam::d{12}:role/TestRole';
+
+program
+    .version(config.version)
+    .description('creates a new IAM Trust role')
+    .option('-n, --rolename [rolename]', 'the name of the role, ' + roleNameDesc)
+    .option('-t, --roletype [roletype]', 'the role type, to see available roles: Cross Account')
+    .option('-T, --trustarn [trustarn]', 'trust arn, ' + trustArnDesc)
+    .option('-a, --account [alksAccount]', 'alks account to use')
+    .option('-r, --role [alksRole]', 'alks role to use')
+    .option('-F, --favorites', 'filters favorite accounts')
+    .option('-v, --verbose', 'be verbose')
+    .parse(process.argv);
+
+var ROLE_NAME_REGEX = /^[a-zA-Z0-9!@+=._-]+$/g,
+    TRUST_ARN_REGEX = /arn:(aws|aws-us-gov):iam::\d{12}:role\/?[a-zA-Z_0-9+=,.@-_/]+/g,
+    roleName        = program.rolename,
+    roleType        = program.roletype,
+    trustArn        = program.trustarn,
+    alksAccount     = program.account,
+    alksRole        = program.role,
+    filterFaves     = program.favorites || false;
+
+utils.log(program, logger, 'validating role name: ' + roleName);
+if(_.isEmpty(roleName) || !ROLE_NAME_REGEX.test(roleName)){
+    utils.errorAndExit('The role name provided contains illegal characters. It must be ' + roleNameDesc);
+}
+
+utils.log(program, logger, 'validating role type: ' + roleType);
+if(_.isEmpty(roleType)){
+    utils.errorAndExit('The role type is required');
+}
+
+utils.log(program, logger, 'validating trust arn: ' + trustArn);
+if(_.isEmpty(trustArn) || !TRUST_ARN_REGEX.test(trustArn)){
+    utils.errorAndExit('The trust arn provided contains illegal characters. It must be ' + trustArnDesc);
+}
+
+if(!_.isUndefined(alksAccount) && _.isUndefined(alksRole)){
+    utils.log(program, logger, 'trying to extract role from account');
+    alksRole = utils.tryToExtractRole(alksAccount);
+}
+
+Iam.getIAMKey(program, logger, alksAccount, alksRole, false, filterFaves, function(err, key, developer){
```

Comment:
The callback has a fourth argument which is their password.

Reply:
`alks-rest-api` is failing when you `POST` `userid & password` along with `aws tokens`